src/utils.zig

```diff
@@ -1408,12 +1408,13 @@ pub const Side = enum {
 	server,
 };
 
+pub const endian: std.builtin.Endian = .big;
```

Comment:
In general I think that all use cases of having this publicly would be better off with just using `.big` directly, since they are either external protocols, where it's better to be explicit in my opinion, or obsolete code that will be removed soon.

I would prefer having this private to avoid needlessly bloating this namespace.